src/App.js

```diff
@@ -1,13 +1,81 @@
 import React from 'react';
+import ReactDOM from "react-dom";
+
+import TodoForm from './components/TodoForm';
+import TodoList from "./components/TodoList";
+
+import "./components/Todo.css";
+
+const tasks =[
+  {
+    name: 'Organize Garage',
+    id: 1528817077286,
+    completed: false
+  },
+  {
+    name: 'Bake Cookies',
+    id: 1528817084358,
+    completed: false
+  }
+];
 
 class App extends React.Component {
-  // you will need a place to store your state in this component.
-  // design `App` to be the parent component of your application.
-  // this component is going to take care of state, and any change handlers you need to work with your state
+  constructor() {
+    super();
+    this.state = {
+      tasks
+    };
+  }
+
+  toggleCompleted = taskId => {
```

Comment:
How could you go about making it clear to the user whether a task is marked as completed or not? Consider using checkboxes or using the text-decoration property "line-through."